ios/RNTPTDocumentView.m

```diff
@@ -4623,6 +4623,9 @@ - (NSString *) getDocumentPath {
 
 - (NSArray<NSDictionary *> *)getAllFieldsForDocumentViewTag:(int)pageNumber
 {
+    if(pageNumber == 0){
```

Comment:
Does this need to be changed to -1 given your other changes?

Reply:
yes changed